deployment/functionapp.bicep

```diff
@@ -159,30 +173,57 @@ resource funcApp 'Microsoft.Web/sites@2021-02-01' = {
         }
         {
           name: 'AzureBlobStorageAccConnectionString'
-          value: '@Microsoft.KeyVault(VaultName=${keyVaultName};SecretName=${storageConnectionStringSecretName})'
-        }
-        {
-          name: 'AzureKeyVaultEndpoint'
-          value: keyvault.properties.vaultUri
+          value: storageConnectionStringValue
         }
+        // {
+        //   name: 'AzureBlobStorageAccConnectionString__serviceUri'
+        //   value: storageAccount.properties.primaryEndpoints.blob
+        // }
+        // {
+        //   name: 'AzureBlobStorageAccConnectionString__credential'
+        //   value: 'managedIdentity'
+        // }
+        // {
+        //   name: 'AzureBlobStorageAccConnectionString__clientId'
+        //   value: managedIdentity.properties.clientId
+        // }
         {
           name: 'AzureManagedIdentityClientId'
           value: managedIdentity.properties.clientId
         }
+        {
+          name: 'AzureCosmosDBConnectionString'
+          value: cosmosDB.properties.documentEndpoint
+        }
+        {
+          name: 'AzureDocumentIntelligenceConnectionString'
+          value: documentIntelligence.properties.endpoint
+        }
+        {
+          name: 'AzureOpenAIConnectionString'
+          value: openAi.properties.endpoint
+        }
+        {
+          name: 'AzureOpenAIModelDeployment'
+          value: 'text-embedding-3-large'
+        }
+        {
+          name: 'AzureOpenAIModelDimensions'
+          value: '1536'
```

Comment:
Didn't we parameterize this? Shouldn't we make it a `bicepparam` too then?

Reply:
Missed it somehow during modifications. Updated back to pass down as params.